modules/pulsepointLiteBidAdapter.js

```diff
@@ -1,301 +1,308 @@
-import {createBid} from 'src/bidfactory';
-import {addBidResponse} from 'src/bidmanager';
+/* eslint dot-notation:0, quote-props:0 */
 import {logError, getTopWindowLocation} from 'src/utils';
-import {ajax} from 'src/ajax';
-import {STATUS} from 'src/constants';
-import adaptermanager from 'src/adaptermanager';
+import { registerBidder } from 'src/adapters/bidderFactory';
+
+const NATIVE_DEFAULTS = {
```

Comment:
These could go into `native.js` if they are per the spec (i think they are). 

Doesn't need to change for merge though.